spin/options.go

```diff
@@ -6,7 +6,8 @@ import "github.com/charmbracelet/gum/style"
 type Options struct {
 	Command []string `arg:"" help:"Command to run"`
 
-	ShowOutput   bool         `help:"Show output of command" default:"false" env:"GUM_SPIN_SHOW_OUTPUT"`
+	ShowOutput   bool         `help:"Show output of command after its completion" default:"false" env:"GUM_SPIN_SHOW_OUTPUT"`
```

Comment:
Looks like Maas said you can outright replace ShowOutput with LiveOutput, should be able to merge once that's done!

Reply:
<--The Point--
      Me
Will do!